src/view-backend.c

```diff
@@ -103,8 +103,6 @@ wpe_view_backend_set_input_client(struct wpe_view_backend* backend, const struct
 void
 wpe_view_backend_set_fullscreen_client(struct wpe_view_backend* backend, const struct wpe_view_backend_fullscreen_client* client, void* client_data)
 {
-    assert(!backend->fullscreen_client);
-
```

Comment:
Something that is an issue with all the `*_client` pointers we store in the `wpe_view_backend`: we do not have a way of telling client code when the clients are not needed anymore. A good idea would be to add a `destroy` callback, but of course that is out of the scope of this patch.

Reply:
@yury-s I agree with the change, but could you please add `(nullable)` annotations to the parameters in the documentation comment in `include/wpe/view-backend.h`, lines 106-107, before merging this? They need to be `(transfer none) (nullable)`.